streaming/src/main/scala/org/apache/spark/streaming/scheduler/StreamingListener.scala

```diff
@@ -66,6 +69,9 @@ case class StreamingListenerReceiverStopped(receiverInfo: ReceiverInfo)
 @DeveloperApi
 trait StreamingListener {
 
+  /** Called when the streaming starts */
```

Comment:
I do mean the start of Streaming
although it pass the startTime of the StreamingContext
this event called soon after `state = StreamingContextState.ACTIVE`
what is the definition of "start of streaming"?